proposals/NNNN-multipart-kit.md

```diff
@@ -0,0 +1,108 @@
+# MultipartKit
+
+* Proposal: [SSWG-NNNN](NNNN-multipart-kit.md)
+* Authors: [Siemen Sikkema](https://github.com/siemensikkema), [Tim Condon](https://github.com/0xTim)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
+* Implementation: [MultipartKit](https://github.com/vapor/multipart-kit)
+* Forum Threads: [Pitch](https://forums.swift.org/t/multipartkit)
```

Comment:
(once this is merged in I will create a forums thread, and then update the link here)